templates/cluster-template.yaml

```diff
@@ -189,7 +195,9 @@ spec:
           kubeletExtraArgs:
             cloud-provider: external
           name: '{{ ds.meta_data["local_hostname"] }}'
-      preKubeadmCommands: []
+      preKubeadmCommands:
+      - echo '10.0.0.100   ${CLUSTER_NAME}-${APISERVER_LB_DNS_SUFFIX}.${AZURE_LOCATION}.cloudapp.azure.com'
```

Comment:
I don't want to over-complicate things, but would there ever be a reason for a user to want to customize the internal IP? If so, it might be worth having it be customizable with an environment variable. Otherwise it should be fine as-is.

Reply:
There is that option in the SPEC for private lbs. This would be a good improvement, but for now I wouldn't block on it.